app/models/form/lettings/pages/housing_provider.rb

```diff
@@ -0,0 +1,32 @@
+class Form::Lettings::Pages::HousingProvider < ::Form::Page
+  def initialize(id, hsh, subsection)
+    super
+    @id = "housing_provider"
+    @header = ""
+    @description = ""
+    @subsection = subsection
+  end
+
+  def questions
+    @questions ||= [
+      Form::Lettings::Questions::HousingProvider.new(nil, nil, self),
+    ]
+  end
+
+  # For an organisation owns and manages ONLY their own housing stock AND NOT uses agents to manage the properties AND NOT manages housing stock of other organisations?
+  # In "set up this lettings log" no extra question should be shown
```

Comment:
Took me a while to understand what this comment meant, maybe something like this is clearer:

#No extra question should be shown in "set up this lettings log" if:
#the organisation owns and manages ONLY their own housing stock
#AND doesn't have managing agents